### ISSUE TITLE
Bug: Fullscreen not working

### ISSUE THREAD

**Issue by TiddlyTweeter** [Jermolene/TiddlyDesktop#148]
Just noting the bug with "fullscreen" not working in the current version, as discussed here: https://groups.google.com/d/msg/tiddlywiki/jLymP4WvovI/kc2oWX7-BgAJ

**Comment by kookma** [comment]
Yes, I use both 32 and 64bit version of TiddlyDesktop, booth of them do not react to fullscreen button!
Please fix!
